sharktank/conftest.py

```diff
@@ -191,25 +191,25 @@ def pytest_addoption(parser):
     parser.addoption(
         "--google-t5-v1-1-small-f32-model-path",
         type=Path,
-        default="/data/t5/small/google__t5-v1_1-small_f32.gguf",
+        default="/shark-dev/data/t5/small/google__t5-v1_1-small_f32.gguf",
```

Comment:
Changing the defaults is okay, but we can also override these values in the workflow to be explicit. All tests should be runnable on any machine (with enough disk space and compatible hardware). That means that all tests should provide code or documentation for downloading prerequisite files to paths that the tests will look for.

Maybe that means setting `HF_HOME` on the runners to `/shark-dev/data` and having scripts run `huggingface-cli download ...`, maybe it means writing more custom caching code.

Please file an issue (if one hasn't been filed already) to remove all hardcoded paths from source files and set an assignee.